single_offer/src/lib.rs

```diff
@@ -187,6 +213,9 @@ impl SingleOfferTrait for SingleOffer {
     }
 
     fn updt_price(e: Env, admin: Authorization, n: u32, d: u32) {
+        if d == 0 {
+            panic!()
```

Comment:
Can you add text to the panic why?
```rust
panic!("d is zero but cannot be zero")
```

Reply:
https://github.com/stellar/soroban-examples/pull/39